app/assets/javascripts/administrate/components/has_many_form.js

```diff
@@ -1,3 +1,3 @@
 $(function() {
-  $(".form-field--has-many select").selectize({});
+  $(".field-unit--has-many select").selectize({});
```

Comment:
Good point, but I didn’t originally write this JavaScript and didn’t want to change it in a CSS-focused PR…